doc/stdlib/strings.rst

```diff
@@ -219,151 +234,253 @@
+       "March               "
+
+       julia> rstrip(a)
+       "March"
+
+.. function:: startswith(s::AbstractString, prefix::AbstractString)
 
    .. Docstring generated from Julia source
 
-   Returns ``true`` if ``string`` starts with ``prefix``\ . If ``prefix`` is a vector or set of characters, tests whether the first character of ``string`` belongs to that set.
+   Returns ``true`` if ``s`` starts with ``prefix``\ . If ``prefix`` is a vector or set of characters, tests whether the first character of ``s`` belongs to that set.
+
+   .. doctest::
+
+       julia> startswith("JuliaLang", "Julia")
+       true
 
-.. function:: endswith(string, suffix)
+.. function:: endswith(s::AbstractString, suffix::AbstractString)
 
    .. Docstring generated from Julia source
 
-   Returns ``true`` if ``string`` ends with ``suffix``\ . If ``suffix`` is a vector or set of characters, tests whether the last character of ``string`` belongs to that set.
+   Returns ``true`` if ``s`` ends with ``suffix``\ . If ``suffix`` is a vector or set of characters, tests whether the last character of ``s`` belongs to that set.
 
-.. function:: uppercase(string)
+   .. doctest::
+
+       julia> endswith("Sunday", "day")
+       true
+
+.. function:: uppercase(s::AbstractString)
 
    .. Docstring generated from Julia source
 
-   Returns ``string`` with all characters converted to uppercase.
+   Returns ``s`` with all characters converted to uppercase.
 
-.. function:: lowercase(string)
+   .. doctest::
+
+       julia> uppercase("Julia")
+       "JULIA"
+
+.. function:: lowercase(s::AbstractString)
 
    .. Docstring generated from Julia source
 
-   Returns ``string`` with all characters converted to lowercase.
+   Returns ``s`` with all characters converted to lowercase.
+
+   .. doctest::
+
+       julia> lowercase("STRINGS AND THINGS")
+       "strings and things"
 
-.. function:: ucfirst(string)
+.. function:: ucfirst(s::AbstractString)
 
    .. Docstring generated from Julia source
 
    Returns ``string`` with the first character converted to uppercase.
 
-.. function:: lcfirst(string)
+   .. doctest::
+
+       julia> ucfirst("python")
+       "Python"
+
+.. function:: lcfirst(s::AbstractString)
 
    .. Docstring generated from Julia source
 
    Returns ``string`` with the first character converted to lowercase.
 
-.. function:: join(strings, delim, [last])
+   .. doctest::
+
+       julia> lcfirst("Julia")
+       "julia"
+
+.. function:: join(io::IO, strings, delim, [last])
 
    .. Docstring generated from Julia source
 
-   Join an array of ``strings`` into a single string, inserting the given delimiter between adjacent strings. If ``last`` is given, it will be used instead of ``delim`` between the last two strings. For example
+   Join an array of ``strings`` into a single string, inserting the given delimiter between adjacent strings. If ``last`` is given, it will be used instead of ``delim`` between the last two strings. For example,
```

Comment:
should reference `io` somehow, c.f. the near-dupe docstring you deleted from io-network.rst
```
Print elements of ``items`` to ``io`` with ``delim`` between them. If ``last`` is specified, it is used as the final delimiter instead of ``delim``\ .
```